### PR TITLE
Allen/sat 1574

### PR DESCRIPTION
**Pull request by Allensy**

### CODE REVIEW

File: dist/examUtility/examUtility.js [hunk 1]
```diff
@@ -1,204 +0,0 @@
-(function (angular) {
-    'use strict';
-
-    angular.module('znk.infra-satsm.examUtility',[]);
-})(angular);
-
-(function () {
```
Comment (assafshp) [inline dist/examUtility/examUtility.js]: why did you delete it ?

File: src/components/completeExerciseSatsm/directives/completeExerciseSummary/completeExerciseSummary.directive.js [hunk 2]
```diff
@@ -145,14 +145,14 @@
                 }
 
                 return function () {
-                    var exerciseCategoryId = $ctrl.completeExerciseCtrl.getExerciseContent().categoryId;
+                    var exerciseCategory = $ctrl.completeExerciseCtrl.getExerciseContent();
```
Comment (Ariel-W) [inline src/components/completeExerciseSatsm/directives/completeExerciseSummary/completeExerciseSummary.directive.js]: add .categoryId;

File: src/components/completeExerciseSatsm/directives/completeExerciseSummary/completeExerciseSummary.directive.js [hunk 3]
```diff
@@ -145,14 +145,14 @@
...
 
                     var TEST_SCORE = 2;
 
                     $ctrl.testScoreMastery = {};
                     $q.all([
                         StatsSrv.getLevelStats(TEST_SCORE),
-                        CategoryService.getCategoryLevel2Parent(exerciseCategoryId)
+                        CategoryService.getCategoryLevel1Parent(exerciseCategory)
```
Comment (Ariel-W) [inline src/components/completeExerciseSatsm/directives/completeExerciseSummary/completeExerciseSummary.directive.js]: use getCategoryLevel1ParentById

File: src/components/configSatsm/config/scoringConfig.js [hunk 4]
```diff
@@ -1,66 +1,28 @@
...
-                    });
-                    return totalScores;
+        ScoringServiceProvider.setScoringLimits({
+            exam: {
+                9: {
+                    min: 250,
+                    max: 800
+                },
+                10: {
+                    min: 300,
```
Comment (Ariel-W) [inline src/components/configSatsm/config/scoringConfig.js]: Use previous implementation

File: src/components/configSatsm/config/scoringConfig.js [hunk 5]
```diff
@@ -1,66 +1,28 @@
...
-        //         },
-        //         10: {
-        //             min: 300,
-        //             max: 800
-        //         }
-        //     }
-        // });
-
-
-        ScoringServiceProvider.setExamScoreFnGetter(function () {
```
Comment (Ariel-W) [inline src/components/configSatsm/config/scoringConfig.js]: Make sure no one uses this...

File: src/components/completeExerciseSatsm/directives/completeExerciseSummary/completeExerciseSummary.directive.js (hunk 2 above)
Comment (choroshin) [inline src/components/completeExerciseSatsm/directives/completeExerciseSummary/completeExerciseSummary.directive.js]: added

File: src/components/completeExerciseSatsm/directives/completeExerciseSummary/completeExerciseSummary.directive.js (hunk 3 above)
Comment (choroshin) [inline src/components/completeExerciseSatsm/directives/completeExerciseSummary/completeExerciseSummary.directive.js]: done

File: src/components/configSatsm/config/scoringConfig.js (hunk 4 above)
Comment (choroshin) [inline src/components/configSatsm/config/scoringConfig.js]: done

File: dist/examUtility/examUtility.js (hunk 1 above)
Comment (choroshin) [inline dist/examUtility/examUtility.js]: we use use `znk.infra-satsm.examUtility` from infra-web-app
